resources/i18n/ptbr/fdmprinter.def.json.po

```diff
@@ -0,0 +1,5178 @@
+"da parede mais externa e a velocidade de preenchimento."
+
+#: fdmprinter.def.json
+msgctxt "speed_topbottom label"
+msgid "Top/Bottom Speed"
+msgstr "Velocidade Superior/Inferior"
+
+#: fdmprinter.def.json
+msgctxt "speed_topbottom description"
+msgid "The speed at which top/bottom layers are printed."
+msgstr "Velocidade em que as camadas superiores e inferiores são impressas."
+
+#: fdmprinter.def.json
+msgctxt "speed_support label"
+msgid "Support Speed"
+msgstr "Velocidade do Suporte"
+
+#: fdmprinter.def.json
+msgctxt "speed_support description"
+msgid ""
+"The speed at which the support structure is printed. Printing support at "
+"higher speeds can greatly reduce printing time. The surface quality of the "
+"support structure is not important since it is removed after printing."
+msgstr ""
+"A velocidade em que a estrutura de suporte é impressa. Imprimir o suporte a "
+"velocidades mais altas pode reduzir bastante o tempo de impressão. A "
+"qualidade de superfície das estruturas de suporte não é importante já que "
+"são removidas após a impressão."
+
+#: fdmprinter.def.json
+msgctxt "speed_support_infill label"
+msgid "Support Infill Speed"
+msgstr "Velocidade do Preenchimento do Suporte"
+
+#: fdmprinter.def.json
+msgctxt "speed_support_infill description"
+msgid ""
+"The speed at which the infill of support is printed. Printing the infill at "
+"lower speeds improves stability."
+msgstr ""
+"A velocidade em que o preenchimento do suporte é impresso. Imprimir o "
+"preenchimento em velocidades menores melhora a estabilidade."
+
+#: fdmprinter.def.json
+msgctxt "speed_support_interface label"
+msgid "Support Interface Speed"
+msgstr "Velocidade da Interface de Suporte"
+
+#: fdmprinter.def.json
+msgctxt "speed_support_interface description"
+msgid ""
+"The speed at which the roofs and bottoms of support are printed. Printing "
+"the them at lower speeds can improve overhang quality."
+msgstr ""
+"A velocidade em que o topo e base dos suportes são impressos. Imprimi-lo a "
+"menores velocidade pode melhor a qualidade das seções pendentes."
+
+#: fdmprinter.def.json
+msgctxt "speed_prime_tower label"
+msgid "Prime Tower Speed"
+msgstr "Velocidade da Torre de Purga"
+
+#: fdmprinter.def.json
+msgctxt "speed_prime_tower description"
+msgid ""
+"The speed at which the prime tower is printed. Printing the prime tower "
+"slower can make it more stable when the adhesion between the different "
+"filaments is suboptimal."
+msgstr ""
+"A velocidade em que a torre de purga é impressa. Imprimir a torre de purga "
+"mais lentamente pode torná-la mais estável quando a aderência entre os "
+"diferentes filamentos é subótima."
+
+#: fdmprinter.def.json
+msgctxt "speed_travel label"
+msgid "Travel Speed"
+msgstr "Velocidade de Viagem"
+
+#: fdmprinter.def.json
+msgctxt "speed_travel description"
+msgid "The speed at which travel moves are made."
+msgstr ""
+"Velocidade em que ocorrem os movimentos de viagem (movimentação do extrusor "
+"s
```

Comment:
Hmm, I think Github isn't happy with these long files. This comment was meant to be placed at the translation of "Anti Overhang Mesh".